### PR TITLE
IA-1941: Adding deployment management features; bump security policy for TLS; update simple example

### PR DESCRIPTION
**Pull request by bushong1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v6.1.0
+
+* **NEW FEATURES**
+  * `enable_deployment_rollbacks` - allows for rollbacks if a container fails to reach a steady state
+  * `wait_for_steady_state` - Makes terraform wait until the ECS Service has fully deployed to a stead state before progressing
```
Comment (yaythecloud) [inline CHANGELOG.md:7]: typo in stead

File: examples/deployment-management/main.tf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module "easy-fargate" {
+  #source  = "USSBA/easy-fargate-service/aws"
```
Comment (yaythecloud) [inline examples/deployment-management/main.tf:2]: want to leave these commented?

**Review by yaythecloud — DISMISSED** [review]
LGTM

File: examples/deployment-management/main.tf (hunk 2 above)
Comment (bushong1) [reply examples/deployment-management/main.tf:2]: yep.  it's helpful as an example

**Review by yaythecloud — APPROVED** [review]
LGTM

**Pull request merged**